python/dgl/backend/tensorflow/tensor.py

```diff
@@ -685,6 +685,9 @@ def grad(x):
 def is_no_grad(x):
     return cgrad.is_no_grad(x)
 
+def is_recording():
+    raise NotImplementedError("Tensorflow doesn't support is_recording")
```

Comment:
Can you give a warning and return True so that tf can still work under train and inference.

Reply:
a few backend APIs don't support tensorflow. we won't be able to support tensorflow for distributed training in 0.5